.gitignore

```diff
@@ -32,3 +32,4 @@ yarn-error.log*
 *.ntvs*
 *.njsproj
 *.sln
+Notas.txt
```

Comment:
I think we should delete this, since it seems more like a personal file

Reply:
Yes, it is a personal file with notes. I put it in git ignore list just not to bother other devs. 
Should I take it away?

Reply:
yea

Reply:
just did in [ca64c76](https://github.com/telosnetwork/app-telos-native/commit/ca64c7610bda65f3d92bed82d50f6256c710131b)